### DISCUSSION TITLE
xdrip drops the connection to Libre 2

### DISCUSSION THREAD

**Discussion by Pipperoo169** [NightscoutFoundation/xDrip#2659] [General]
Hello. My xdrip frequently drops the connection to Libre 2. The connection gap may last up to 30 mins. When I scan Libre 2 with Librelink patched app, the connection restores. Is it acceptable or should there be some setup to avoid this?

**Comment by Navid200** [comment]
I suggest that you choose a title for the discussions that best represents the issue.
If the issue is about maintaining connectivity to Libre, the title stating that data does not upload to Nightscout has no relevance.

**Comment by Pipperoo169** [comment]
Hi,
Sorry, but I changed the title to relevant. Not sure why it’s still about uploading data to nightscout…

Отправлено с iPhone

**Comment by Navid200** [comment]
@psonnera

**Reply by Pipperoo169** [reply]
Probably I didn't press save button to change the title...Just an idea :)

**Comment by psonnera** [comment]
If LibreLink loses connection to the sensor, xDrip+ won't show data. Please confirm you have alarms in Librelink when this occurs. Please note Android 13 is a major issue. If your phone upgraded, this is unfortunately expected but in no way an xDrip+ issue.

**Comment by Pipperoo169** [comment]
I have switched off LibreLink connection notification. I’ll switch it on and see how it goes.
I have Android 10, so I assume this shouldn’t be an issue.

Thanks for advice!

Отправлено с iPhone
